mode/examples/Basics/Color/WaveGradient/WaveGradient.pyde

```diff
@@ -1,11 +1,11 @@
 """
-Wave Gradient 
-by Ira Greenberg.    
+Wave Gradient
+by Ira Greenberg.
 
 Generate a gradient along a sin() wave.
 """
 amplitude = 30
-fillGap = 2.5
+fillGap = 2
```

Comment:
This was the original reason for the change. This is used later on in a call to range(). Python requires parameters to range() be integers. So the original would not even run for me.